### DISCUSSION TITLE
align action buttons

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#3695] [Help] [answered]
is it possible to align the action buttons to the right instead of the left?

**Comment by danharrin** [comment, marked as answer]
Which action buttons?

**Reply by jcgmalave** [reply]
Create and Cancel, below the form

**Reply by danharrin** [reply]
It's in the config file `layout.forms.actions.alignment`

**Reply by jcgmalave** [reply]
thanks a lot!!!
